SwaggerThemes/SwaggerTheme.cs

```diff
@@ -34,17 +28,16 @@ public static string GetSwaggerThemeCss(Theme theme)
     /// <param name="app">The web application to which the Swagger UI is added.</param>
     /// <param name="theme">The theme to be applied to the Swagger UI.</param>
     /// <param name="customStyles">Optional custom styles to be applied to the Swagger UI.</param>
-    public static void UseSwaggerThemes(this WebApplication app, Theme theme, string? customStyles = null)
+    /// <param name="setupAction">Additional swagger setup options</param>
+    public static void UseSwaggerUi(
```

Comment:
I capitalized the I to be consistent with the upstream method name and show up in overload resolution with the upstream methods.